app/src/main/java/com/codesoom/models/SendResponseData.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.models;
+
+import com.codesoom.HttpEnum.HttpStatusCode;
+
+/**
+ *  SendResponseData 클래스는 HttpExchange sendResponseHeaders 메소드에 들어가는 매개변수를 담고 있는 클래스입니다.
```

Comment:
너무 좋은 학습 자료를 만들어 주셨어요. 자 이 주석에는 `SendResponseData`라는 자기 자신의 이름도 들어가 있습니다. 그런데 이 클래스의 이름이 `ResponseData`로 바뀐다면 어떻게 될까요? 주석도 같이 `ResponseData`로 수정해줘야겠죠?

주석을 왜 수정해줘야 할까요? 바뀌기 전 클래스 이름이 주석에 계속 남아있게 되면 거짓 사실을 알려주는 주석이 되어 버리기 때문입니다. 그래서 주석을 바꾸기 싫어도 바꿔줘야 하는 것입니다. 이 문제를 프로그래밍에서는 "주석이 클래스 이름에 의존하고 있다"고 표현합니다. 불필요한 의존이 있기 때문에 아주 작은 변경이 있는데도 변경이 암세포처럼 주석까지 전이되는 것입니다.

그렇다면 이 주석은 어떻게 작성해야 좋을까요?



```suggestion
 *  SendResponseData 클래스는 HttpExchange sendResponseHeaders 메소드에 들어가는 매개변수를 담고 있는 클래스입니다.
```

Reply:
왜 저는 이런 생각을 못하였는지 ..ㅎㅎ!!  잘못된 주석 하나로 개발자에게 많은 번거로움이 따라올 수도 있다는 것을 알게 되었습니다 !
주석은 최대한 간단 명료한 설명으로 나타내도록 작성하는 연습을 꾸준하게 해야할 것 같습니다.

"HttpExchange sendResponseHeaders의 매개변수를 담는 클래스" 로 수정했습니다!

Reply:
`HttpExchange`도 아직 남아있잖아요?!

Reply:
헉.. 자바에 기본으로 제공하는 클래스도 예외가 아니였군요...! 더 신중하게 작성 해보도록 하겠습니다!